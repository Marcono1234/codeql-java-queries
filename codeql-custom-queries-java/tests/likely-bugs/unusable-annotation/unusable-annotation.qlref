./likely-bugs/unusable-annotation.ql
