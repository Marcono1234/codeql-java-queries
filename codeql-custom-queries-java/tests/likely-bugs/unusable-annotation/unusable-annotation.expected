| Test.java:7:16:7:23 | Unusable | Annotation type is not usable |
